pyboy/plugins/game_wrapper_tetris.py

```diff
@@ -110,7 +114,11 @@ def start_game(self):
                 break
 
         # Start game. Just press Start when the game allows us.
-        for _ in range(3):
+        for i in range(3):
+            if i == 2:
+                PyBoyGameWrapper._set_timer_div(self, timer_div, randomize_div)
```

Comment:
This could be called earlier. For example from `PyBoyGameWrapper.start_game(self, timer_div=timer_div, randomize_div=randomize_div)`, as that's what `reset_game` does.

It just need to be set at some point before `save_state`.

Reply:
Did it like that so that both start_game and reset_game would give the same exact state when called, as there will be more ticks in start_game. Not sure if this is really needed or not, but thought it would be good to let the user have the option of having the exact same value each time, if they provide a specific value for the divider. It could be moved before the loop, where the state could also be saved. Then have three button presses in reset game instead of one. Not sure how much this will impact performance though. PyBoyGameWrapper.start_game gives a warning if called after frame 0, which is why I did not use it. Do let me know if you have a better idea.